Artifacts/windows-certificate-from-keyvault/Artifactfile.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Can you please format the file? Parameters is missing some indentation to make it easier to read.